base/server/src/main/java/com/netscape/cms/servlet/csadmin/UpdateNumberRange.java

```diff
@@ -268,16 +268,14 @@ protected void process(CMSRequest cmsReq) throws EBaseException {
             // insert info
             logger.debug("UpdateNumberRange: Sending response");
 
-            // send success status back to the requestor
-            XMLObject xmlObj = new XMLObject();
-            Node root = xmlObj.createRoot("XMLResponse");
-
-            xmlObj.addItemToContainer(root, "Status", SUCCESS);
-            xmlObj.addItemToContainer(root, "beginNumber", beginNum.toString(radix));
-            xmlObj.addItemToContainer(root, "endNumber", endNum.toString(radix));
-            byte[] cb = xmlObj.toByteArray();
-
-            outputResult(httpResp, "application/xml", cb);
+            // send success status back to the requester
+            JSONObject jsonObj = new JSONObject();
+            ObjectNode responseNode = jsonObj.getMapper().createObjectNode();
+            responseNode.put("Status", SUCCESS);
+            responseNode.put("beginNumber", beginNum.toString(radix));
+            responseNode.put("endNumber", endNum.toString(radix));
+            jsonObj.getRootNode().set("Response", responseNode);
```

Comment:
Will the root changing from `XMLResponse` to just `Response` possibly break something somewhere

Reply:
Yeah it does, but fortunately the CI fails spectacularly if you don't find all the linked places in the code - making a `JSONObject` out of `XMLObject` results in sadness. In this example, the roots have the same name, so it is fine. Any new code would have to use the new root name.